embedded-graphics/src/image/mod.rs

```diff
@@ -29,7 +176,7 @@ pub trait ImageFile<'a>: Dimensions + Sized {
     fn height(&self) -> u32;
 }
 
-pub use self::image16bpp::Image16BPP;
-pub use self::image1bpp::Image1BPP;
-pub use self::image8bpp::Image8BPP;
+pub use self::image16bpp::{Image16BPP, ImageType16BPP};
+pub use self::image1bpp::{Image1BPP, ImageType1BPP};
+pub use self::image8bpp::{Image8BPP, ImageType8BPP};
```

Comment:
Do these `*Type*` types need to be publicly exported? I'd rather not expand the API surface if possible.

Reply:
If `ImageType` is change to be internal these souldn't be exposed either.

Reply:
Got it, let's make it internal then :slightly_smiling_face: